### PR TITLE
v2.15.1 Adding limit_response to get_tasks method

### PR DESCRIPTION
**Pull request by FernandoLopez85**
# Pull Request Summary

## Description

Added the limited_response argument to the get_tasks() method

## How did you test your code?

_Which of the following have you done to test your changes? Please describe the tests that you ran to verify your changes._
- [ ] Created new unit tests in `tests/` for the newly implemented methods
- [x] Updated existing unit tests in `tests/` to cover changes made to existing methods

## Checklist

_Please make sure all items in this checklist have been fulfilled before sending your PR out for review!_

- [x] I have commented my code in details, particularly in hard-to-understand areas
- [x] I have updated [Readme.rst](https://github.com/scaleapi/scaleapi-python-client/blob/master/README.rst) document with examples for newly implemented public methods
- [ ] I have reviewed [Deployment and Publishing Guide for Python SDK](https://github.com/scaleapi/scaleapi-python-client/blob/master/docs/pypi_update_guide.md) document
- [x] I incremented the SDK version in `_version.py` _(unless this PR only updates the documentation)._
- [ ] In order to release a new version, a "Release Summary" needs to be prepared and published after the merge

### CODE REVIEW

File: scaleapi/__init__.py [hunk 1]
```diff
@@ -273,6 +273,9 @@ def tasks(self, **kwargs) -> Tasklist:
                 If true, returns a pre-signed s3 url for the
                 attachment used to create the task.
 
+            limited_response (bool):
+                If true, returns limited task response.
```
Comment (jonathanfeng-scale) [inline scaleapi/__init__.py:277]: can we document what fields are returned in the limited_response?

File: scaleapi/__init__.py [hunk 2]
```diff
@@ -392,6 +397,9 @@ def get_tasks(
...
```
Comment (jonathanfeng-scale) [inline scaleapi/__init__.py:401]: probably document it here too, just so people know in the future what it is instead of asking us 😆

**Review by jonathanfeng-scale — APPROVED** [review]
If we can add in a test-case to make sure that the limited_response is working (fields that shouldn't be there don't exist), that'll be perfect!

File: scaleapi/__init__.py (hunk 1 above)
Comment (fatihkurtoglu) [reply scaleapi/__init__.py:277]: Agree, we can both document it in this comment section as well as document it in the Readme with more details.

Comment (FernandoLopez85) [reply scaleapi/__init__.py]: Got it! Just updated with the new version specifying fields to be returned as comments and also added to the readme

**Review by ivan-scale — APPROVED** [review]
LGTM

**Review by lautarorangil — APPROVED** [review]

File: README.rst [hunk 3]
```diff
@@ -171,12 +171,14 @@ Accessing ``task.params`` child objects directly at task level is **deprecated**
     task.params["geometries"]   # task.geometries is DEPRECATED
     task.params["attachment"]   # task.attachment is DEPRECATED
 
+If you use the ``limited_response = True`` filter in ``get_tasks()``, you will receive the following attributes: ``_id``, ``status``, ``metadata``, ``project``, ``otherVersion``.
+
```
Comment (fatihkurtoglu) [inline README.rst:175]:
```suggestion
If you use the ``limited_response = True`` filter in ``get_tasks()``, you will only receive the following attributes: ``task_id``, ``status``, ``metadata``, ``project`` and ``otherVersion``.

```

File: scaleapi/__init__.py [hunk 4]
```diff
@@ -392,6 +398,10 @@ def get_tasks(
                 If true, returns a pre-signed s3 url for the
                 attachment used to create the task.
 
+            limited_response (bool):
+                If true, returns task response of the following fields:
+                _id, status, metadata, project, otherVersion.
```
Comment (fatihkurtoglu) [inline scaleapi/__init__.py:403]:
```suggestion
                task_id, status, metadata, project, otherVersion.
```

File: scaleapi/__init__.py [hunk 5]
```diff
@@ -273,6 +273,10 @@ def tasks(self, **kwargs) -> Tasklist:
...
```
Comment (fatihkurtoglu) [inline scaleapi/__init__.py:278]:
```suggestion
                task_id, status, metadata, project, otherVersion.
```

**Review by fatihkurtoglu — APPROVED** [review]

**Pull request merged**